packages/dashboard/package.json

```diff
@@ -29,9 +29,14 @@
     "update-snapshots": "node ../../scripts/build.js jest -u"
   },
   "devDependencies": {
+    "@types/enzyme": "3.1.5",
```

Comment:
When you change dependencies, re-run `npm run generate` from the root to regenerate the shrinkwrap.

Reply:
Also try to keep the versions consistent. (e.g. use whatever version of enzyme is used elsewhere, or, update them all.) reduces overhead and potential issues resulting from mismatched versions.

Reply:
I made sure all the versions are matching with other packages :)

Reply:
Have you ran rush generate?

Reply:
Just did, will merge now. Thanks!